setup.sh

```diff
@@ -1,49 +1,58 @@
 #!/usr/bin/env bash
 #encoding=utf8
 
+function check_installed_pip() {
+   ${PYTHON} -m pip > /dev/null
+   if [ $? -ne 0 ]; then
+        echo "pip not found. Please make sure that pip is available for ${PYTHON}."
```

Comment:
If we are more specific here, this may help the user installing the bot identify the problem and add a dependency required:
"pip not found (called as '${PYTHON} -m pip')."